packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -96,11 +95,15 @@ class TimelineScreen extends Screen {
           ]),
         div()..flex(),
       ]);
-    upperButtonSection.add(getServiceExtensionButtons());
+    upperButtonSection.add(
+      div(c: 'btn-group collapsible')
+        ..add(<CoreElement>[
+          ServiceExtensionButton(performanceOverlay).button,
+        ]),
+    );
```

Comment:
nit: can we just add the performanceOverlay button directly in the creation of upperButtonSection?

Reply:
done